content/en/blog/2021/announcing-etcd-3.5.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Announcing etcd 3.5
+date: 2021-06-15
```

Comment:
@hexfusion @ptabor Do we have any release date in mind? (We can change it later as well)

Reply:
6/15 seems reasonable given the current trajectory +1